src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsLanguageDebugInfo.cs

```diff
@@ -96,7 +96,7 @@ public int GetNameOfLocation(IVsTextBuffer pBuffer, int iLine, int iCol, out str
                                 if (nullablePoint.HasValue)
                                 {
                                     var point = nullablePoint.Value;
-                                    var document = point.Snapshot.GetOpenDocumentInCurrentContextWithChanges();
+                                    var document = CodeAnalysis.Text.Extensions.GetDocument(point.Snapshot);
```

Comment:
What happened here?

Reply:
looks like complexify/simplify was unhappy for some reason!